### PR TITLE
Add features pages

### PR DESCRIPTION
**Pull request by elv-zenia**

### CODE REVIEW

File: src/main/static/stylesheets/features.scss [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+        gap: 8px;
+        margin-bottom: 12px;
+
+        &__details {
+          font-size: var(--font-size-body-5);
+        }
+      }
+
+      &__list {
+        &__item {
```
Comment (elv-kevin) [inline src/main/static/stylesheets/features.scss:125]: try not to nest items this deeply (this is for .tenancy-info__features__content__list__item)

you don't need every child to be nested, you can have .tenancy-info__features, .tenancy-info__content, .tenancy-info__list, .tenancy-info__items.

I'm trying to stick to 1 nesting in most cases, occasionally 2 if it really warrants it. 

if something within a component is more complex and needs nesting, consider just giving it a different parent name

e.g. .component__sub-component__child -> .sub-component__child

File: src/main/pages/features/TenancyInfo.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  addedBenefitText="",
+  monthlyPrice,
+  additionalCostText
+}) => {
+  return (
+    <div className="tenancy-info">
+      <div className="tenancy-info__level">{ level }</div>
+      <h4 className="tenancy-info__tenancy">{ tenanciesMap[tenancy] }</h4>
+      <div className="tenancy-info__service">{ service }</div>
+      <hr />
```
Comment (elv-kevin) [inline src/main/pages/features/TenancyInfo.js:32]: never heard of the hr tag, interesting

File: src/main/pages/features/Features.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+              content: TenanciesList({monthly: false})
+            }
+          ]}
+        />
+
+        <div className="features-connect">
+          <Button className="light primary features-connect__button">Connect with us</Button>
+        </div>
+
+        <div className="features-banner">
```
Comment (elv-kevin) [inline src/main/pages/features/Features.js:95]: The banner and the faq are supposed to be full-bleed according to the design. I alluded to this when we talked, but I think what we can do is in MainApp.js -> PageContainer, we can add parameters for before/after content that is not within the padded block.

```
const PageContainer = observer(({children, before, after, padded=false, dark=false, unbound=false}) => {
 ....

  return (
    <>
      <Header />
      { before }
      <div
        key={`page-container-${location.pathname}`}
        className={`page-container fade-in ${padded ? "padded-block" : ""} ${dark ? "dark" : "light"} ${unbound ? "unbound" : ""}`}
      >
        {children}
      </div>
      { after }
      <ContactForm dark={dark} />
      <Footer dark={dark} />
    </>
  );
});
```

alternatively, and actually I think this is a better idea, specify this page as unpadded, then add 'padded-block' div around the stuff on the page that's supposed to be padded.

File: src/main/pages/technology/Technology.js [hunk 4]
```diff
@@ -53,21 +53,27 @@ const Technology = () => {
               title: "Architecture",
               icon: TechnologyIcons.ArchitectureIcon,
               content: (
-                <ExpandableImage expandable image={ArchitectureDiagram} imageClassName="component-test__image" />
+                <div className="tabbed-info-box--shadow">
```
Comment (elv-kevin) [inline src/main/pages/technology/Technology.js:56]: I think the proper class for this is .tabbed-info-box__content--shadow, or use a more general .shadow class

.tabbed-info-box--shadow implies that this is a variant of tabbed-info-box, but it's tabbed-info-box__content.

File: src/main/pages/features/TenancyInfo.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import ImageIcon from "../../components/ImageIcon";
+import {CheckmarkIcon} from "../../static/icons/Icons";
```
Comment (elv-kevin) [inline src/main/pages/features/TenancyInfo.js:3]: this is unused. run `npm run lint-main` to find stuff like this

File: src/main/static/stylesheets/variables.scss [hunk 6]
```diff
@@ -24,6 +24,8 @@ $width-mobile: 600px;
   --color-highlight-1: #c07cff;
   --color-highlight-2: #9b51e0;
   --color-highlight-3: #791cd0;
+  --color-highlight-4: #efecf4;
```
Comment (elv-kevin) [inline src/main/static/stylesheets/variables.scss:27]: these are both like white, they shouldn't be highlights.

Comment (elv-kevin) [reply src/main/static/stylesheets/variables.scss:27]: also seems like it's being used as a background, use one of the --background variables instead

File: src/main/static/stylesheets/variables.scss [hunk 7]
```diff
@@ -55,7 +59,7 @@ $width-mobile: 600px;
 
   --background-gradient-footer-light: linear-gradient(180deg, rgba(155, 81, 224, 0) 0%, rgba(155, 81, 224, 0.1) 100%);
   --background-gradient-footer-dark: linear-gradient(180deg, rgba(41, 17, 64, 0) 0%, rgba(41, 17, 64, 0.5) 100%);
-
+  --background-gradient-banner: linear-gradient(180deg, #471671 0%, #371870 100%);
```
Comment (elv-kevin) [inline src/main/static/stylesheets/variables.scss:62]: name this --background-gradient-features-banner, since it's specific to that banner

**Review by elv-kevin — CHANGES_REQUESTED** [review]

File: src/main/pages/features/Features.js (hunk 3 above)
Comment (elv-zenia) [reply src/main/pages/features/Features.js:95]: Haha I actually was only adding the padding in a media query for tablet breakpoint but I'll add `padded-block` to the html for those elements

Comment (elv-zenia) [reply src/main/pages/features/Features.js:95]: Oh I didn't notice the `max-width: 1600px` on the `page-container`. I'll just add the full width components to `after`

**Comment by elv-kevin** [comment]
this is looking great. just a couple things I noticed
- the design has the individual items inline at desktop width, (e.g name  description  'view doc'). this can break down for smaller screens (just set flex-direction: column and maybe change the gap size)
- reduce the padding more on smaller screens in the accordions
- the design has all of the features on the same page. can use a hash param and IDs to scroll to the right section from a link

File: src/main/static/stylesheets/components.scss [hunk 8]
```diff
@@ -267,3 +267,21 @@
...
 }
+
+.accordion {
+  padding: 40px 20px 40px 50px;
+
+  &__header {
+    display: flex;
+
+    &__button {
+      margin-left: auto;
```
Comment (elv-kevin) [inline src/main/static/stylesheets/components.scss:278]: I think this is better expressed as 

&__header {
  display: flex;
  justify-content: space-between;
}

**Pull request merged**